src/pages/Contributing/GitConventions/index.md

```diff
@@ -1,4 +1,10 @@
 ---
```

Comment:
I think we should do all of this in one page, there is a lot of repetitions about conventional commits, by removing them, we should be able to have one page that is not too big